panels/shopping-list/ha-panel-shopping-list.html

```diff
@@ -79,7 +80,7 @@
             <paper-listbox slot="dropdown-content">
               <paper-item
                 on-tap="_clearCompleted"
-              >Clear completed</paper-item>
+              >[[haLocalize('ui.panel.shopping-list', 'clear_completed')]]</paper-item>
```

Comment:
Does it make sense to prefix it with ui? Doesn't panel already imply that?

Reply:
What I was envisioning is `ui` would be a top-level container for all frontend text that is simply used for display purposes, and isn't semantically linked to anything on the backend. The panel namespace here would contain the text we plan to fragment off into different files, but there would also likely be `ui` text that belongs in the core data set if it's used in more-info displays, etc. This was partially also driven by the fact that `panel` is already a top level key containing our panel names.

That being said this was just my first attempt at this. Open to suggestions or other proposals here.

Reply:
Makes sense.